lib/src/main/java/uk/gov/di/ipv/cri/kbv/api/domain/Alerts.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.di.ipv.cri.kbv.api.domain;
+
+import java.util.List;
+
+public class Alerts {
+
+    List<String> alerts;
```

Comment:
This could be confusing, Class name and field name are the Same except for the Uppercase

Reply:
Agreed, I have added the `@JsonProperty`